### PR TITLE
DOC-672 Bloblang playground

### PR DESCRIPTION
**Pull request by JakeSCahill**
## Description

This PR introduces a new interactive Bloblang playground, allowing users to experiment with Bloblang expressions in a user-friendly UI environment. The playground enables real-time Bloblang experiments, supported by a WebAssembly backend for performance and cross-browser compatibility.

Users can input JSON data, define mappings, and view the output directly in the playground interface. We also provide sample mappings that users can view and try out.

User inputs are saved to local storage so that they can leave the playground to view the reference docs and come back to continue.

The WebAssembly backend is included in a [prerelease of our UI project](https://github.com/redpanda-data/docs-ui/releases/tag/v2.1.0-alpha). As part of the bundle process, we build the `blobl.wasm` file from a modified version of this[ Go code](https://github.com/mihaitodor/benthos-playground/blob/main/wasm/blobl-editor/cmd/wasm/main.go) from @mihaitodor . (The source of our modified version is in our UI project in the `blobl-editor/` directory).

**How to test**

1. [Open the Bloblang playground](https://deploy-preview-98--redpanda-connect.netlify.app/redpanda-connect/guides/bloblang/playground/).
2. Select a sample.
3. See the result.

## Page previews

[Preview](https://deploy-preview-98--redpanda-connect.netlify.app/redpanda-connect/guides/bloblang/playground/)

## Checks

- [x] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

**Comment by rockwotj** [comment]
This is awesome!

File: local-antora-playbook.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ content:
     branches: main
 ui:
   bundle:
-    url: https://github.com/redpanda-data/docs-ui/releases/latest/download/ui-bundle.zip
+    url: ../docs-ui/build/ui-bundle.zip
```
Comment (JakeSCahill) [inline local-antora-playbook.yml:21]:
```suggestion
    url: https://github.com/redpanda-data/docs-ui/releases/download/v2.1.0-alpha/ui-bundle.zip
```

File: local-antora-playbook.yml [hunk 2]
```diff
@@ -18,7 +18,7 @@ content:
     branches: main
 ui:
   bundle:
-    url: https://github.com/redpanda-data/docs-ui/releases/latest/download/ui-bundle.zip
+    url: https://github.com/redpanda-data/docs-ui/releases/download/v2.1.0-alpha/ui-bundle.zip
```
Comment (Deflaimun) [inline local-antora-playbook.yml:21]: reminder to change back

**Review by Deflaimun — APPROVED** [review]
Looks amazing.

**Pull request merged**